package/kedro_viz/integrations/kedro/sqlite_store.py

```diff
@@ -5,18 +5,18 @@
 import json
 import logging
 from pathlib import Path
-from typing import Any, Generator, Type
+from typing import Any, Generator
 
 from kedro.framework.session.store import BaseSessionStore
-from sqlalchemy.orm.session import Session
+from sqlalchemy.orm import sessionmaker
 
 from kedro_viz.database import create_db_engine
 from kedro_viz.models.experiments_tracking import Base, RunModel
 
 logger = logging.getLogger(__name__)
 
 
-def get_db(session_class: Type[Session]) -> Generator:
+def get_db(session_class: sessionmaker) -> Generator:
```

Comment:
I guess this answers my question about whether `sessionmaker` was the right type before 😄